.github/workflows/ci.yml

```
@@ -106,3 +106,126 @@ jobs:
+      - name: "Check out the repo"
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: "Install Pnpm"
+        uses: pnpm/action-setup@v2
+        with:
+          version: "8"
+
+      - name: "Setup Node.js"
+        uses: actions/setup-node@v3
```

actionlint (1.7.9)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)